mysql-test/t/threads_running_gtid-slave.opt

```diff
@@ -0,0 +1 @@
+--gtid-mode=ON --enforce_gtid_consistency=ON
```

Comment:
Remove this file as `--gtid-mode=ON --enforce_gtid_consistency=ON` options will be already present in `my.cnf` of the `rpl_gtid` suite.